.github/workflows/pull-request-checks.yml

```diff
@@ -10,7 +10,7 @@ env:
 jobs:
   block-fixup:
     name: Block fixup commits
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
```

Comment:
Just trivial thought. I found some jobs run on `ubuntu-latest`. I can understand specifying the specific version for the jobs that requires build, but for example, this `block-fixup`, `Filter changes`, and maybe more, I don't think it has to be on the specific OS version. 
Anyway it's too trivial and I don't think it's worth changing. If we want consistency, changing all `ubuntu-latest` to `ubuntu-24.04` might not be a bad idea.

Reply:
Yeah I left at least the jobs which are marked as ubuntu-latest as that generally indicates that the runner environment does not matter so match, and things should work if the underlying runner image changes. We'll do a review of the runner types in a different PR to see if we can consolidate, but not a huge priority right now.